store/src/main/java/com/nytimes/android/external/store3/base/impl/RealInternalStore.java

```diff
@@ -79,7 +80,7 @@
     public Observable<Parsed> getRefreshing(@Nonnull final Key key) {
         return get(key)
                 .toObservable()
-                .compose(StoreUtil.<Parsed, Key>repeatWhenCacheEvicted(refreshSubject, key));
+                .compose(StoreUtil.<Parsed, Key>repeatWhenSubjectEmits(refreshSubject, key));
```

Comment:
We get an `Observable`, we convert it to a `Single` and then back to `Observable`?